<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Agent 007</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="SpyLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern No. 20</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Agent007&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="BoxesLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern No. 20</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Boxes:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="GoalLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>210</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern No. 20</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Goal:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>290</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern No. 20</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Score:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="BoxesValue">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>130</y>
      <width>51</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="GoalValue">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>220</y>
      <width>41</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreValue">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>308</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BoxesPercent">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;0%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="GoalPercent">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>220</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;0%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>460</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 85, 0);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
    <property name="iconSize">
     <size>
      <width>38</width>
      <height>38</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="RulesWindow">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1291</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI Variable Small Light'; font-size:26pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="source">
     <url>
      <string>file:///C:/Users/Виктор/Desktop/Университет/4семестр/Питон/1Аттестация/MineSweeper/MineSweeper/MineSweeper/Rules.html</string>
     </url>
    </property>
   </widget>
   <widget class="QTableWidget" name="field">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>38</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>38</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>38</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>38</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>38</number>
    </attribute>
   </widget>
   <zorder>SpyLabel</zorder>
   <zorder>BoxesLabel</zorder>
   <zorder>GoalLabel</zorder>
   <zorder>ScoreLabel</zorder>
   <zorder>BoxesValue</zorder>
   <zorder>GoalValue</zorder>
   <zorder>ScoreValue</zorder>
   <zorder>BoxesPercent</zorder>
   <zorder>GoalPercent</zorder>
   <zorder>ResetButton</zorder>
   <zorder>field</zorder>
   <zorder>RulesWindow</zorder>
  </widget>
  <widget class="QMenuBar" name="MenuForWindow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="FieldMenu">
     <property name="title">
      <string>Параметры поля</string>
     </property>
     <addaction name="SmallFieldAction"/>
     <addaction name="MediumFieldAction"/>
     <addaction name="LargeFieldAction"/>
    </widget>
    <addaction name="FieldMenu"/>
   </widget>
   <widget class="QMenu" name="Rules">
    <property name="title">
     <string>Правила</string>
    </property>
    <addaction name="ShowAction"/>
    <addaction name="HideAction"/>
   </widget>
   <addaction name="Help"/>
   <addaction name="Rules"/>
  </widget>
  <action name="SmallFieldAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Маленькое (10х10), 5 мин</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="MediumFieldAction">
   <property name="text">
    <string>Среднее (15х15), 7 мин</string>
   </property>
  </action>
  <action name="LargeFieldAction">
   <property name="text">
    <string>Большое (20х20), 10 мин</string>
   </property>
  </action>
  <action name="ShowAction">
   <property name="text">
    <string>Показать правила игры</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="HideAction">
   <property name="text">
    <string>Скрыть правила игры</string>
   </property>
  </action>
  <action name="actionfh">
   <property name="text">
    <string>fh</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>HideAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>RulesWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ShowAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>RulesWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
